packages/next/client/image.tsx

```diff
@@ -642,6 +642,18 @@ export default function Image({
     }
   }
 
+  let imageSrcSetPropName = 'imageSrcSet'
+  let imageSizesPropName = 'imageSizes'
+  if (!process.env.__NEXT_REACT_ROOT) {
```

Comment:
Should we inverse this so that we don't include `if (true) { ... }` for all non-react root projects?